generic_config_updater/services_validator.py

```diff
@@ -3,15 +3,41 @@
 
 logger = genericUpdaterLogging.get_logger(title="Service Validator")
 
+print_to_console = False
+
+def set_verbose(verbose=False):
+    global print_to_console, logger
+
+    print_to_console = verbose
+    if verbose:
+        logger.set_min_log_priority_debug()
+    else:
+        logger.set_min_log_priority_notice()
+
+
 def _service_restart(svc_name):
-    os.system(f"systemctl restart {svc_name}")
-    logger.log_notice(f"Restarted {svc_name}")
+    rc = os.system(f"systemctl restart {svc_name}")
+    logger.log(logger.LOG_PRIORITY_NOTICE,
+            f"Restarted {svc_name}", print_to_console)
+    return rc == 0
 
 
 def ryslog_validator(old_config, upd_config, keys):
-    _service_restart("rsyslog-config")
+    return _service_restart("rsyslog-config")
 
 
 def dhcp_validator(old_config, upd_config, keys):
-    _service_restart("dhcp_relay")
+    return _service_restart("dhcp_relay")
+
+
+def vlan_validator(old_config, upd_config, keys):
+    old_vlan = old_config.get("VLAN", {})
+    upd_vlan = upd_config.get("VLAN", {})
+
+    for key in set(old_vlan.keys()).union(set(upd_vlan.keys())):
+        if (old_vlan.get(key, {}).get("dhcp_servers", []) != 
+                upd_vlan.get(key, {}).get("dhcp_servers", [])):
+            return _service_restart("dhcp_relay")
```

Comment:
NICE! We just restart dhcp at the first occurrence of mismatch between dhcp_servers in old vs updated.